terminatorlib/keybindings.py

```diff
@@ -103,27 +107,25 @@ def _parsebinding(self, binding):
             raise KeymapError('No key found')
         keyval = Gdk.keyval_from_name(key)
         if keyval == 0:
-            raise KeymapError("Key '%s' is unrecognised" % key)
+            raise KeymapError(f"Key '{key}' is unrecognised")
```

Comment:
Why are you changing this string formatting?

Reply:
F-strings have been preferred since Python 3.6 went GA.